client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-plugin-install/index.tsx

```diff
@@ -0,0 +1,79 @@
+import { StepContainer } from '@automattic/onboarding';
+import { useI18n } from '@wordpress/react-i18n';
+import { useEffect } from 'react';
+import wpcomRequest from 'wpcom-proxy-request';
+import DocumentHead from 'calypso/components/data/document-head';
+import { LoadingEllipsis } from 'calypso/components/loading-ellipsis';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import type { Step } from '../../types';
+
+const SiteMigrationPluginInstall: Step = function WaitForAtomic( { navigation, data } ) {
```

Comment:
We should not use two function names to describe the same function, it will cause issue to read the stack trace

Reply:
Good catch! That's another copy/paste thing I missed.